WooCommerce/Classes/POS/Models/PointOfSaleAggregateModel.swift

```diff
@@ -95,10 +99,20 @@ protocol PointOfSaleAggregateModelProtocol {
         self.collectOrderPaymentAnalyticsTracker = collectOrderPaymentAnalyticsTracker
         self.searchHistoryService = searchHistoryService
         self.paymentState = paymentState
+        self.popularItemsController = popularItemsController
+
         publishCardReaderConnectionStatus()
         publishPaymentMessages()
         setupReaderReconnectionObservation()
     }
+
+    var popularItemsState: [POSItemType: [POSItem]] {
+        popularItemsController.itemsByType
```

Comment:
Should `popularItemsController` be non-private and allow the views to access it directly?

Reply:
Possibly. I went back and forth on it and kind of punted the decision to later by taking the more conservative path. 

I'm not sure whether we want the architecture to go all the way down that path, even though we've done it for the search vs coupons vs products controllers. I think it'll depend on the design of the reusable component.